TLM/TLM/Util/PriorityRoad.cs

```diff
@@ -149,65 +291,74 @@ public static class PriorityRoad {
         ///  - split avenue into 2 oneway roads
         ///  - semi-roundabout
         /// </summary>
-        public static void FixJunction(ushort nodeId) {
+        public static void FixHighPrioirtyJunction(ushort nodeId) {
             if (nodeId == 0) {
                 return;
             }
 
-            List<ushort> segmentList = new List<ushort>();
+            List<ushort> nodeSegments = new List<ushort>();
             for (int i = 0; i < 8; ++i) {
                 ushort segId = nodeId.ToNode().GetSegment(i);
                 if (segId != 0) {
-                    segmentList.Add(segId);
+                    nodeSegments.Add(segId);
                 }
             }
 
-            if (segmentList.Count < 3) {
+            if (nodeSegments.Count < 3) {
                 Log._Debug("FixJunction: This is not a junction.");
```

Comment:
Same note. Who will be reading this message, and how to figure out its location? It is quite short, and reading log files to support some user will tell us nothing.